lib/src/widgets/dashboard_screen/summary.dart

```diff
@@ -23,132 +23,212 @@ class Summary extends StatelessWidget {
+                                    .where((iter) => iter.maturity * 100 < 100)
                                     .toList()
-                                    .length),
+                                    .length,
+                                SarakaColors.darkGray.withOpacity(0.2),
+                              ),
+                            ],
+                            outsideLabelStyleAccessorFn: (MatureCount row, _) =>
+                                charts.TextStyleSpec(
+                                    color: charts.MaterialPalette.black),
+                          ),
+                        ],
+                        animate: true,
+                      ),
+                    ),
+                    Column(
+                      crossAxisAlignment: CrossAxisAlignment.center,
+                      children: <Widget>[
+                        Row(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: <Widget>[
+                            Text(
+                              ((cardsMaturity
+                                              .where((iter) =>
+                                                  iter.maturity * 100 >= 100)
+                                              .toList()
+                                              .length /
+                                          totalCardsNumber) *
+                                      100)
+                                  .toStringAsFixed(1)
+                                  .toString(),
+                              overflow: TextOverflow.ellipsis,
+                              style: SarakaTextStyles.heading
+                                  .copyWith(fontSize: 48),
+                            ),
+                            Column(
+                              children: <Widget>[
+                                Padding(
+                                  padding: EdgeInsets.only(bottom: 20),
+                                ),
+                                Text(
+                                  '%',
+                                  overflow: TextOverflow.ellipsis,
+                                  style: SarakaTextStyles.heading
+                                      .copyWith(fontSize: 24),
+                                ),
+                              ],
+                            ),
+                          ],
+                        ),
+                        Row(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: <Widget>[
+                            Padding(
+                              padding: EdgeInsets.only(right: 24),
+                            ),
+                            Icon(
+                              Icons.fiber_manual_record,
+                              size: 12,
+                              color: SarakaColors.lightRed,
+                            ),
```

Comment:
Use just `Container` and border radius instead

Reply:
@axross Instead of icon?

Reply:
Yes

Reply:
Done.